subsys/bluetooth/controller/ll_sw/pdu.h

```diff
@@ -516,6 +516,11 @@ enum pdu_data_llctrl_type {
 	PDU_DATA_LLCTRL_TYPE_UNUSED = 0xFF
 };
 
+enum pdu_data_cte_present {
+	PDU_DATA_CTE_PRESENT_BIT_DISABLED = 0x0,
+	PDU_DATA_CTE_PRESENT_BIT_ENABLED = 0x1
+};
```

Comment:
This looks like it could just be a boolean value instead

Reply:
enum removed, I'll use 'true` or `false` instead.